oxidation/libparsec/crates/protocol/tests/test_pki.rs

```diff
@@ -0,0 +1,207 @@
+    //   cmd: "pki_enrollment_accept"
+    //   device_certificate: hex!("3c64756d6d793e")
+    //   enrollment_id: ext(2, hex!("e89621b91f8e4a7c8d3182ee513e380f"))
+    //   redacted_device_certificate: hex!("3c64756d6d793e")
+    //   redacted_user_certificate: hex!("3c64756d6d793e")
+    //   user_certificate: hex!("3c64756d6d793e")
+    &hex!(
+        "89ae6163636570745f7061796c6f6164c4073c64756d6d793eb86163636570745f7061796c"
+        "6f61645f7369676e6174757265c40b3c7369676e61747572653ebd61636365707465725f64"
+        "65725f783530395f6365727469666963617465c41f3c61636365707465725f6465725f7835"
+        "30395f63657274696669636174653ea3636d64b5706b695f656e726f6c6c6d656e745f6163"
+        "63657074b26465766963655f6365727469666963617465c4073c64756d6d793ead656e726f"
+        "6c6c6d656e745f6964d80256f48ed307984f10830e197287399c22bb72656461637465645f"
+        "6465766963655f6365727469666963617465c4073c64756d6d793eb972656461637465645f"
+        "757365725f6365727469666963617465c4073c64756d6d793eb0757365725f636572746966"
+        "6963617465c4073c64756d6d793e"
+    )[..],
+    authenticated_cmds::AnyCmdReq::PkiEnrollmentAccept(authenticated_cmds::pki_enrollment_accept::Req {
+        accept_payload: hex!("3c64756d6d793e").to_vec(),
+        accept_payload_signature: hex!("3c7369676e61747572653e").to_vec(),
+        accepter_der_x509_certificate: hex!("3c61636365707465725f6465725f783530395f63657274696669636174653e").to_vec(),
+        device_certificate: hex!("3c64756d6d793e").to_vec(),
+        enrollment_id: EnrollmentID::from_hex("56f48ed307984f10830e197287399c22").unwrap(),
+        redacted_device_certificate: hex!("3c64756d6d793e").to_vec(),
+        redacted_user_certificate: hex!("3c64756d6d793e").to_vec(),
+        user_certificate: hex!("3c64756d6d793e").to_vec()
+    })
+)]
+#[case::list(
+    // Generated from Python implementation (Parsec v2.14.0)
+    // Content:
+    //   cmd: "pki_enrollment_list"
+    &hex!("81a3636d64b3706b695f656e726f6c6c6d656e745f6c697374")[..],
+    authenticated_cmds::AnyCmdReq::PkiEnrollmentList(authenticated_cmds::pki_enrollment_list::Req { })
+)]
+#[case::reject(
+    // Generated from Python implementation (Parsec v2.14.0)
+    // Content:
+    //   cmd: "pki_enrollment_reject"
+    //   enrollment_id: ext(2, hex!("88a75cc10b8d43d9b1f91ad8a12be8ee"))
+    &hex!(
+      "82a3636d64b5706b695f656e726f6c6c6d656e745f72656a656374ad656e726f6c6c6d656e"
+      "745f6964d80288a75cc10b8d43d9b1f91ad8a12be8ee"
+    )[..],
+    authenticated_cmds::AnyCmdReq::PkiEnrollmentReject(authenticated_cmds::pki_enrollment_reject::Req {
+        enrollment_id: EnrollmentID::from_hex("88a75cc10b8d43d9b1f91ad8a12be8ee").unwrap()
+    })
+)]
+fn serde_authenticated_pki_req(
+    #[case] bytes: &[u8],
+    #[case] expected_serde_result: authenticated_cmds::AnyCmdReq,
+) {
+    let data = authenticated_cmds::AnyCmdReq::load(bytes).expect("Failed to load raw req");
+
+    assert_eq!(data, expected_serde_result);
```

Comment:
I don't understand why you call it `serde_result`, since it isn't a `load` from the `serde` crate neither a `serialization`